QStreak/View Controllers/View Record/RecordDetailViewController.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  RecordDetailViewController.swift
+//  QStreak
+//
+//  Created by Daniel McCoy on 4/9/20.
+//  Copyright © 2020 Rocket Insights. All rights reserved.
+//
+
+import UIKit
+
+class RecordDetailViewController: UIViewController {
+
+    // MARK: - Outlets
+    @IBOutlet weak var recordDateLabel: UILabel!
+
+    @IBOutlet weak var contactCountLabel: UILabel!
+
+    @IBOutlet weak var activityLabel: UILabel!
+
+    // MARK: - Properties
+
+    var recordToEdit: Record?
```

Comment:
Just call this `record` (Can right click and hit `Refactor -> Rename`...does all the work for ya 😄) . Also, this should be in the view model not in the controller.